source/shared/cpp/ObjectModel/Enums.h

```diff
@@ -347,7 +347,10 @@ namespace AdaptiveSharedNamespace
         Good,
         Attention,
         Warning,
-        Accent
+        Accent,
+        // this is different from None, None is a style while NotSet isn't a style
```

Comment:
>None is a style [](start = 40, length = 15)

Are you sure "None" is a style? I'm thought we were using None to mean not-set already....

Reply:
none is a style in a sense that it has no attribute.
Not Set means value has not been initialized.

---
In reply to: [266177514](https://github.com/Microsoft/AdaptiveCards/pull/2544#discussion_r266177514) [](ancestors = 266177514)

Reply:
I'm sorry, i'm having trouble grokking this. So, we use none when the style has not been set. Saying that it "is a style" seems inaccurate (there's no host config for the "None" style, for example). If None is used when the style is not set, what is not-set for and in what case do we need to distinguish between them?

---
In reply to: [266181375](https://github.com/Microsoft/AdaptiveCards/pull/2544#discussion_r266181375) [](ancestors = 266181375,266177514)

Reply:
I think we could maybe just use None for this? Looks like the only place it's being used is as a return from GetParentalContainerStyles which is otherwise returning a value from m_parentalContainerStyles. But it also looks like you're never putting the None style into m_parentalContainterStyles, so returning None there could mean unset without ambiguity. Let me know if I'm missing something.

---
In reply to: [266523140](https://github.com/Microsoft/AdaptiveCards/pull/2544#discussion_r266523140) [](ancestors = 266523140,266181375,266177514)